README.adoc

```diff
@@ -1,6 +1,6 @@
 // Do not edit this file (e.g. go instead to src/main/asciidoc)
```

Comment:
Helps if I read the comments. Where does the Lombok change go then? I only see it in the root README.adoc